packages/ember/src/components/mktg/horizontal-card.gts

```diff
@@ -0,0 +1,54 @@
+import Component from '@glimmer/component';
+import Card from '../card.gts';
+
+interface HorizontalCardSignature {
+  Element: HTMLDivElement;
+  Args: {
+    title?: string;
+    price?: string;
+  };
+  Blocks: {
+    left: [];
+    right: [];
+  };
+}
+
+export default class HorizontalCard extends Component<HorizontalCardSignature> {
+  get title() {
+    return this.args.title;
+  }
+  get price() {
```

Comment:
What do you think about renaming this to something more generic? I'm just thinking about the "what if" we wanted to use a card that outlined speed - 1000/Mbps instead of $65/month

Reply:
That would require removing the /mo as well so styling would have to be done on the front end (i.e. the span that makes the /mo smaller and aligned differently than the number). If you think that's a better route to go, I can change that pretty quickly!

Reply:
@TSenter any thoughts on this?

Reply:
Addressed and refactored to callout named block